doofinder/includes/api/class-management-api.php

```diff
@@ -132,7 +132,7 @@ private function sendRequest( $url, $data ) {
 	public function updateItem( $item_id, $name, $data ) {
 		$this->log->log( 'Update item' );
 
-		return $this->sendRequest( "$this->api_host/api/v2/search_engines/$this->hash/indices/$name/items/$item_id", array(
+		return $this->sendRequest( $this->api_host."api/v2/search_engines/$this->hash/indices/$name/items/$item_id", array(
```

Comment:
The string "api/v2/search_engines/" is repeated a lot of times. Extract to a constant?